src/intrinsics/ecma262/Object.js

```diff
@@ -165,6 +166,10 @@ export default function(realm: Realm): NativeFunctionValue {
           temporalTo.values = new ValuesDomain(to);
         }
         to.temporalAlias = temporalTo;
+        // Store the args for the temporal so we can easily clone
+        // and reconstruct the temporal at another point, rather than
+        // mutate the existing temporal
+        realm.temporalAliasArgs.set(temporalTo, temporalArgs);
```

Comment:
Can we just expose them as a property on the temporal so we don't have to remember to set them in a global map?

Reply:
We can do, but I thought this way was better because we're not introducing another property onto AbstractObjectValue.

Reply:
@gaearon Do you feel that we should move to using properties? I'm happy to make this change now, but I don't have any strong preference. I actually thought of doing that originally but then thought feedback would be "can't you move this into a Map, rather than adding another property". Heh.

Reply:
Do you expect that there will be more places where we want to reconstruct a temporal? Or is this more React-specific?

I don't care strongly either way. I think I'd like to have a property so we don't have to remember to track them. But if it's only relevant for our props objects then meh,

Reply:
It's only for our props. If we have other cases, I'll move it out in a follow up PR. :)